### ISSUE TITLE
[jsk_apc2016_common] Build error due to lack of cmake dependencies

### ISSUE THREAD

**Issue by pazeshun** [start-jsk/jsk_apc#1541]
I got the following error in `catkin build`
```
Scanning dependencies of target sib_topic_synchronizer
[  6%] Building CXX object CMakeFiles/sib_topic_synchronizer.dir/src/sib_topic_synchronizer.cpp.o
/home/ushun/apc2016/apc_ws/src/start-jsk/jsk_apc/jsk_apc2016_common/src/sib_topic_synchronizer.cpp:5:54: fatal error: jsk_apc2016_common/SegmentationInBinSync.h: 
 #include <jsk_apc2016_common/SegmentationInBinSync.h>
                                                      ^
compilation terminated.
make[2]: *** [CMakeFiles/sib_topic_synchronizer.dir/src/sib_topic_synchronizer.cpp.o]  1
make[1]: *** [CMakeFiles/sib_topic_synchronizer.dir/all]  2
make: *** [all]  2
[jsk_apc2016_common] <== '/home/ushun/apc2016/apc_ws/build/jsk_apc2016_common/build_env.sh /usr/bin/make --jobserver-fds=3,5 -j' failed with return code '2'
                                                                                                                                                 
Failed   <== jsk_apc2016_common         [ 12.0 seconds ]                 
```
What is `SegmentationInBinSync.h`? I couldn't find that file.

**Comment by pazeshun** [comment]
@yuyu2172

**Comment by pazeshun** [comment]
I got it.
I'll fix this

**Comment by pazeshun** [comment]
`SegmentationInBinSync` is ROS message defined in jsk_apc2016_common package, and `sib_topic_synchronizer.cpp` uses that message.
But `sib_topic_synchronizer.cpp` has no cmake dependency on jsk_apc2016_common, so error occurs.
Similar problem occurs in `cloud_to_spatial_features.cpp` too.
They are fixed in #1543
